.scalafmt.conf

```diff
@@ -0,0 +1,4 @@
+version = 2.0.0
+maxColumn = 160
```

Comment:
160 characters seems like a lot considering [80 characters is the general rule of thumb](https://www.google.com/search?q=how+many+characters+per+line+of+code&sa=X&ved=0ahUKEwjIxL7dksflAhWQr54KHTU9BB4Q7xYILSgA&biw=1680&bih=916). I think the team consensus during the idea exchange was 120 characters, which seems like a good in-between.

Reply:
Yea I started with 120, but had a lot fewer lines of code change with 160. I'll go with consensus though, standby.